### PR TITLE
Add case for obj rest name equal to a property

### PR DESCRIPTION
**Pull request by leobalter**
cc @caiolima

### CODE REVIEW

File: src/dstr-assignment/obj-rest-same-name.case [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: >
+    Proper setting in the values for rest name equal to a property name.
+template: default
+esid: pending
+includes: [propertyHelper.js]
```
Comment (caiolima) [inline src/dstr-assignment/obj-rest-same-name.case]: I think you don't need this.

Comment (leobalter) [inline src/dstr-assignment/obj-rest-same-name.case]: you're right, thanks!

Comment (leobalter) [inline src/dstr-assignment/obj-rest-same-name.case]: There are more cases I repeat this, I'll fix all of them!

**Comment by rwaldron** [comment]
LGTM

**Comment by leobalter** [comment]
Somehow I forgot to push the fixes for this.

**Comment by rwaldron** [comment]
I've actually just reviewed this twice (accidentally), once before a meeting and then again after the meeting.

LGTM.

**Comment by leobalter** [comment]
Thank you! Thank you! :)

**Pull request merged**